src/runtime/plan/plan-consumer.ts

```diff
@@ -89,12 +89,12 @@ export class PlanConsumer {
                !!this.arc.activeRecipe.handles.find(activeHandle => activeHandle.id === handle.id);
       });
       const usesRemoteNonRootSlots = suggestion.plan.slots.find(slot => {
-        return !slot.name.includes('root') && !slot.tags.includes('root') &&
+        return slot.name && !slot.name.includes('root') && !slot.tags.includes('root') &&
```

Comment:
sorry, i wasn't paying enough attention yesterday, i thought the null slot name is happening sometime during planning. 
i need to look into this, i don't think slot name should be null in a resolved recipe (it may cause different sorts of weird stuff in other places). 
could you file an issue, or reply with a recipe that ends up like this? I guess this is the recipe in this PR, then I should patch and try it.

Reply:
I'll look into the test failure, I probably broke it messing with descriptions.

I'll remove the plan-consumer.ts patch and see if the error still comes up. Yes, these are the recipes that triggered the error.

Will get back to you, thanks for the notes!